src/main/java/org/ships/vessel/sign/MoveSign.java

```diff
@@ -36,9 +36,9 @@
 
     public final List<AText> SIGN = Arrays.asList(
             AText.ofPlain("[Move]").withColour(NamedTextColours.YELLOW),
-            AText.ofPlain("{Engine}}"),
             AText.ofPlain(""),
-            AText.ofPlain(2 + "")
+            AText.ofPlain("Speed"),
```

Comment:
The "{Engine}" can evoke that you can select something else - but you can't - that's why i changed in to "Speed" - it makes more sense to me (and also describes what the number means).